blueprints/component/index.js

```diff
@@ -15,7 +15,6 @@ const OCTANE = has('octane');
 // TODO: this should be reading from the @ember/canary-features module
 // need to refactor broccoli/features.js to be able to work more similarly
 // to https://github.com/emberjs/data/pull/6231
```

Comment:
Assume this should be removed as well?

Reply:
sorry for the confusion.  I am new to open source